src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java

```diff
@@ -657,4 +661,15 @@ private ContextMenu createSettingsPopup() {
 
         return contextMenu;
     }
+
+    private void addAlwaysCitedOnPagesTextOption(ContextMenu contextMenu) {
+        OpenOfficePreferences openOfficePreferences = preferencesService.getOpenOfficePreferences();
+
+        CheckMenuItem alwaysAddCitedOnPagesText = new CheckMenuItem(Localization.lang("Automatically add \"Cited on pages...\" at beginning of citation"));
+
+        alwaysAddCitedOnPagesText.selectedProperty().set(openOfficePreferences.getAlwaysAddCitedOnPages());
+        alwaysAddCitedOnPagesText.setOnAction(e -> openOfficePreferences.setAlwaysAddCitedOnPages(alwaysAddCitedOnPagesText.isSelected()));
+
+        contextMenu.getItems().add(alwaysAddCitedOnPagesText);
```

Comment:
I'm really embarrassed I forgot about the CStyles, will add the removal mechanism and move the item below the "old" option as requested.

Reply:
> I'm really embarrassed I forgot about the CSL Styles, will add the removal mechanism and move the item below the "old" option as requested.

Please be comfortable, you are new to the codebase, so that's very normal!